internal/otgutils/print.go

```diff
@@ -30,6 +31,7 @@ import (
 // LogFlowMetrics displays the otg flow statistics.
 func LogFlowMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	t.Helper()
+	time.Sleep(1 * time.Second)
```

Comment:
Assuming there's a good reason for the sleep, can we add a comment here? (Or alternatively, is there a way to add a reasonable Await?)

Reply:
Basically before every loss Pct calculation we're printing the stats using these otgutils functions and we want this short delay in order to make sure there are no packets in transit while polling. Added comments